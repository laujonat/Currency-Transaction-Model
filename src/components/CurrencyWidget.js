import React, { Component } from 'react';

export default class CurrencyWidget extends Component {
  constructor() {
    super();
    this.hi = "hi";
  }

  render() {
    return (<div>"hi"</div>);
  }
}
