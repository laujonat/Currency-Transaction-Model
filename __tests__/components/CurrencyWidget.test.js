import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import configureStore from 'redux-mock-store'; // Smart components

import CurrencyWidget from '../../src/components/CurrencyWidget';


describe('<CurrencyWidget />', () => {
  describe('render()', () => {
    test('renders currency widget', () => {
      const wrapper = shallow(<CurrencyWidget />).dive();
      console.log(wrapper);
      expect(wrapper.exists().to.equal(false));
    });
  });

});
