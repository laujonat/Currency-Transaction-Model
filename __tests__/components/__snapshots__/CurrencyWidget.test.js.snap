exports[`<CurrencyWidget /> render() renders currency widget 1`] = `
<div>
  \"hi\"
</div>
`;
